librz/type/type.c

```diff
@@ -673,3 +673,34 @@ RZ_API char *rz_type_db_get_struct_member(RzTypeDB *typedb, RZ_NONNULL const cha
 	free(members);
 	return res;
 }
+
+RZ_API RZ_OWN RzList /* RzTypeStructMember */ *rz_type_db_struct_member_list(RzTypeDB *typedb, RZ_NONNULL const char *type) {
```

Comment:
You should be able to just use `rz_type_db_get_base_type()` instead of introducing a new function.

Reply:
Good catch, indeed it's easier and shorter. Done.